app/models/encryptable/credentials.rb

```diff
@@ -11,15 +12,25 @@ class Encryptable::Credentials < Encryptable
   validates :name, length: { maximum: 70 }
   validates :name, uniqueness: { scope: :folder }
   validates :folder_id, presence: true
+  # validates :cleartext_custom_attr, presence: { if: :cleartext_custom_attr_label }
```

Comment:
not required anymore? commented?

Reply:
as we discussed already ... if no label set, just assume a default like `custom`. so you don't have to validate it

Reply:
We validate it in frontend right now, should I also remove it?